samples/hello-kapt/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+import org.gradle.api.internal.HasConvention
+import org.gradle.kotlin.dsl.delegateClosureOf
+import org.jetbrains.kotlin.gradle.plugin.KaptAnnotationProcessorOptions
+import org.jetbrains.kotlin.gradle.plugin.KaptJavacOptionsDelegate
+import org.jetbrains.kotlin.gradle.plugin.KotlinSourceSet
+
+plugins {
+    application
+    idea
+    kotlin("jvm") version "1.1.61"
+    kotlin("kapt") version "1.1.61"
+}
+
+application {
+    mainClassName = "samples.HelloAutoValueKt"
+}
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    compile(kotlin("stdlib"))
+    testCompile("junit:junit:4.12")
+
+    //Use AutoValue to check that annotation processing works
+    //https://github.com/google/auto/tree/master/value
+    compileOnly("com.google.auto.value:auto-value:1.5")
+    //Kapt configuration for an annotation processor
+    kapt("com.google.auto.value:auto-value:1.5")
+}
+
+kapt {
+    // Type safe accessors for Kapt DSL
+    correctErrorTypes = true
+
+    // Example of Javac Options configuration
+    javacOptions(delegateClosureOf<KaptJavacOptionsDelegate> {
+        option("SomeJavacOption", "OptionValue")
+    })
+
+    //Kapt Arguments configuration
```

Comment:
:x: this comment and others above miss a space after `//`

Reply:
Fixed